src/FileSearchApplication.java

```diff
@@ -0,0 +1,55 @@
+/*It is the main class where I am taking the filepath as an input from user and checking the presence of seached word*/
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.sql.SQLException;
+import java.util.ArrayList;
+
+public class FileSearchApplication {
+    protected static String contentOfFile, filepath, searchedWord;
+
+
+    public static void main(String[] args) {
+        try {
+            BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
+            System.out.println("enter file path");
```

Comment:
The requirement is to accept command line arguments, not to prompt the user and accept input.
I have already commented that you are not following requirements and to update it!

Please don't waste reviewers time with partial PR updates. Only request review after all comments are properly addressed.

Reply:
Ok sir.

Reply:
Sorry for the mistake.